code/data-tracker/data-tracker.js

```diff
@@ -873,3 +912,16 @@ $(document).on("knack-view-render.view_1252", function (event, page) {
 ////////////////////////////////////////////
 /// End Technician Time Log Validation ///
 ////////////////////////////////////////////
+
+//// Update link text to cabinet details page from signal detals
```

Comment:
Instead of JS, use this [Trigger Button article](https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-LzNSSXajDFpzOv2IhIv/knack-code/looks/trigger-buttons) styling the button with CSS and calling it with the HTML whenever you need it.

Reply:
ah, that's a cool trick! TIL. those docs are excellent!

in this case i'm styling a details link from a connection field, so i don't have control in the builder over what html is rendered by Knack. without this custom code, the link text would simply be the display field on the cabinets record. 

for the CSS, i'm just using the same classes that knack uses on their menu buttons. that actually might be a nice improvement in your docs. you might not need to write any custom CSS if you can get by with the knack classes.